components/community-groups/CommunityGroupsCard.jsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Card, CardContent, Typography, Button, InputLabel, 
+  TextField, FormControl
+} from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import NativeSelect from '@material-ui/core/NativeSelect';
+// EditableCG here
+// import EditableEvent from '../editableEvent/EditableEvent';
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 250,
+  },
+  deleteButton: {

```

Comment:
I assume buttons will be added at some point?

Reply:
Yes, anything that looks like it is not used will be used in a new PR.